aries-site/src/pages/templates/lists.mdx

```diff
@@ -1,16 +1,109 @@
 import { Example } from '../../layouts';
-import { ListScreenExample } from '../../examples';
+import {
+  ListIconIdentifierExample,
+  ListImageIdentifierExample,
+  ListNameOptionActionExample,
+  ListNameActionExample,
+  ListNameDescriptionOptionExample,
+  ListScreenExample,
+} from '../../examples';
 
 HPE Design System list view templates are go-to patterns for displaying homogeneous data such as services, devices, users, and more.
-
-List views are optimized for scanability and reading comprehension. Each list item provides users focussed information and identity labels to aid selection, decision making, and action.
-
-## Manage Users Screen
+List views are optimized for scanability and reading comprehension. Each list item provides users focused information and identity labels to aid selection, decision making, and action.
 
 <Example
   code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/list-views/ListScreenExample.js"
+  docs="https://v2.grommet.io/list?theme=hpe#props"
```

Comment:
Should we have a Figma link to this thumbnail? https://www.figma.com/file/tMYTr17r6hhGajSlNRiPje/HPE-List-Component?node-id=36%3A19